packages/starter/example/components/ContextProvider.tsx

```diff
@@ -12,6 +12,7 @@ import {
     SolflareWalletAdapter,
     TorusWalletAdapter,
     WalletConnectWalletAdapter,
+    SpotWalletAdapter,
```

Comment:
Please limit changes to adding the package only. We don't add all new wallets to the example or starter projects.